src/System.Private.Xml/src/Resources/Strings.resx

```diff
@@ -179,6 +179,9 @@
   </data>
   <data name="Xml_InternalError" xml:space="preserve">
     <value>An internal error has occurred.</value>
+  </data>
+    <data name="Xml_UnreachableLocation" xml:space="preserve">
+    <value>This program location is thought to be unreachable.</value>
```

Comment:
@krwq there is an Xml_InternalError, but the message is different, maybe could be fine, what do you think?

Reply:
I'd go with Xml_InternalError + a comment in the code - many people bucketize with just "works" or "doesn't work" - if we can't provide message which let's them fix the error then I think it is better to skip the details.

Reply:
done